runtime/commonMain/src/kotlinx/serialization/Polymorphic.kt

```diff
@@ -9,13 +9,6 @@ import kotlinx.serialization.internal.*
 import kotlinx.serialization.modules.*
 import kotlin.reflect.*
 
-@Suppress("UNUSED", "DeprecatedCallableAddReplaceWith")
-@Deprecated(
-    message = "Top-level polymorphic descriptor is deprecated, use descriptor from the instance of PolymorphicSerializer or" +
-            "check for descriptor kind instead", level = DeprecationLevel.ERROR
```

Comment:
It's still `WARNING` in the master branch, so leave it be for now.

Reply:
moved to Migrations.kt